ci/scripts/quick-bazel.sh

```diff
@@ -8,7 +8,20 @@
 # This doesn't install dependencies, but should otherwise behave the
 # same as what CI would do on a pull request
 
-bazel query "rdeps(//..., //hw:verilator)" | \
-            sed -e 's/^/-/' | \
-            xargs bazel test --nobuild_tests_only --test_tag_filters=-cw310,-verilator -- //...
+set -e -x
+
+# This queries for things that depend on //hw:verilotor to exclude them
+bazel query 'rdeps(//..., //hw:verilator)' | sed -e 's/^/-/' > bazel-results/slow_tests.txt
```

Comment:
Should this also have a `mkdir -p bazel-results`?

Reply:
Oh yeah, I wonder what that will fix